tested getters and createBoard methods with different size rows and c…

EVWatson opened the pull request:
…olumns

tests all passing

EVWatson committed:
tested getters and createBoard methods with different size rows and columns

tests all passing

jonathan-loe commented on Tic_Tac_Toe_V2.1/test/BoardTest.java:
```diff
@@ -81,7 +81,21 @@ public void methodGetCurrentBoard_whenGivenALargerNumberOfRowsThanColumns_return
-    public void methodCreateBoard_whenGivenNumberOfRowsAndColumns_returnsArrayOfStringsWithCorrectNumnberOfRowsAndColumns(){
+    public void methodCreateBoard_whenGivenNumberOfRowsAndColumns_returnsArrayOfStringsWithCorrectNumberOfRowsAndColumns(){
 
+        Board currentBoard = new Board(6, 6);
```
I understand this test is trying to test `createBoard()` method.

could you pls tell me what `Board currentBoard = new Board(6, 6)` is for? :)

EVWatson replied:
its for testing if the board is still ok if given dimensions other than 3x3

jonathan-loe replied:
sorry Emily - I can't see that in the test (or I don't quite follow you). could you pls elaborate that more? 😞

EVWatson replied:
hmm i think maybe i don't understand what you're asking? might be easier to go over in person tomorrow

EVWatson replied:
oh hang on. you mean, what is Board currentBoard = new Board(6, 6). Its a new instantiation of object Board, with 6 rows and 6 columns given as arguments.

EVWatson committed:
Method to return board as string

if given the same number of rows and columns, will return with correct formatting (spaces etc)
if given an un-even amount of rows and columns, will error out. can't work out why

EVWatson committed:
For method whatIsCurrentBoard

Fixed inner loop to be currentBoard[row].length
Same for if statement involving the column

test now passes when given a differnt number of rows to columns

EVWatson committed:
removed uneccessary test, made create board private again

jonathan-loe commented on Tic_Tac_Toe_V2.1/src/Board.java:
```diff
@@ -12,18 +12,39 @@ public Board(int totalRows, int totalColumns) {
         for (int row = 0; row < newBoard.length; row ++) {
             for (int column = 0; column < newBoard[row].length; column++) {
                 newBoard[row][column] = "*";
+
```
👾

EVWatson committed:
notes from pairing with alex added
